analysers/analyser_merge_school_FR.py

```diff
@@ -81,17 +81,18 @@ def __init__(self, config, logger = None):
             Mapping(
                 select = Select(
                     types = ["nodes", "ways", "relations"],
-                    tags = {"amenity": ["school", "kindergarten"]}),
+                    tags = {"amenity": "school"}),
                 osmRef = "ref:UAI",
                 conflationDistance = 50,
                 generate = Generate(
-                    static2 = {"source": self.source},
+                    static2 = {
+                        "source": self.source,
+                        "amenity": "school"},
                     mapping1 = {
-                        "amenity": lambda res: "kindergarten" if res["Code nature"] in ("101", "102", "103", "111") else "school",
                         "ref:UAI": "Code établissement",
                         "school:FR": lambda res: self.School_FR_nature_uai[res["Code nature"]],
                         "operator:type": lambda res: "private" if res[u"Secteur Public/Privé"] == u"Privé" else "public" if res[u"Secteur Public/Privé"] == u"Public" else None},
-                    mapping2 = {"name": "Appellation officielle"},
+                    mapping2 = {"name": lambda res: res["Appellation officielle"].replace(u"Ecole", u"École").replace(u"ECOLE", u"École").replace(u"College", u"Collège").replace(u"elementaire", u"élémentaire").replace(u"ELEMENTAIRE", u"élémentaire") if res["Appellation officielle"] not in ("A COMPLETER", u"Ecole primaire", u"Ecole Primaire", u"ECOLE PRIMAIRE", u"Ecole PRIMAIRE", u"ECOLE Primaire", u"école primaire", u"École Primaire", u"Ecole primaire publique", u"ECOLE PRIMAIRE PUBLIQUE", u"Ecole Primaire Publique", u"Ecole PRIMAIRE publique", u"Ecole primaire privée", u"ECOLE PRIMAIRE PRIVÉE", u"Ecole primaire intercommunale", u"Ecole primaire Intercommunale", u"Ecole Primaire Intercommunale", u"Ecole élémentaire", u"Ecole Elémentaire", u"Ecole elémentaire", u"Ecole élementaire", u"ECOLE ELEMENTAIRE", u"Ecole ELEMENTAIRE", u"Ecole elementaire", u"Ecole Elementaire", u"École Élémentaire", u"école élémentaire", u"Ecole élémentaire publique", u"Ecole élémentaire Publique", u"ECOLE ELEMENTAIRE PUBLIQUE", u"Ecole élémentaire privée", u"Ecole élémentaire intercommunale", u"Ecole élémentaire école publique", u"Ecole maternelle", u"ECOLE MATERNELLE", u"Ecole Maternelle", u"Ecole MATERNELLE", u"École Maternelle", u"école maternelle", u"Ecole maternelle publique", u"ECOLE MATERNELLE PUBLIQUE", u"Ecole Maternelle Publique", u"Ecole maternelle Publique", u"ecole maternelle publique", u"Ecole maternelle intercommunale", u"Ecole maternelle Intercommunale", u"Collège")},
```

Comment:
Travis fails on this (long) line. Invalid syntax.
Have you been able to run this update analyser ?